configs/config.go

```diff
@@ -29,11 +29,11 @@ type TomlConfig struct {
 	Team                  []TeamConfig
 }
 
-func LoadConfig() TomlConfig {
+func LoadConfig(configFilePath *string) TomlConfig {
```

Comment:
[q] Why use a pointer here, rather than the string itself?

Reply:
Okay, I was mostly wondering why in this particular case. No objections, just curious. I know that Go can get finicky sometimes about pointers, so I tend to approach them very carefully. 😄

Reply:
haha for sure! I think both ways work fine here. Just trying to lean toward the most optimal way.